src/Roslyn.Suppressions/Roslyn.Suppressions/SuppressorCodeGen.targets

```diff
@@ -17,6 +18,10 @@
     <!-- Reference the Sonar analyzers we want to import, but don't include them as analyzers for this project -->
     <PackageReference Include="SonarAnalyzer.CSharp" Version="$(EmbeddedSonarAnalyzerVersion)" ExcludeAssets="all" PrivateAssets="all" GeneratePathProperty="true" />
     <PackageReference Include="SonarAnalyzer.VisualBasic" Version="$(EmbeddedSonarAnalyzerVersion)" ExcludeAssets="all" PrivateAssets="all" GeneratePathProperty="true" />
+
+    <!-- NuGet packages required during the T4 transformation -->
+    <PackageReference Include="Microsoft.CodeAnalysis.Workspaces.Common" Version="3.3.1" PrivateAssets="all" IncludeAssets="none" GeneratePathProperty="true" />
+    <PackageReference Include="System.Composition.AttributedModel" Version="1.0.31" PrivateAssets="all" IncludeAssets="none" GeneratePathProperty="true" />
```

Comment:
The packages are marked with `IncludeAssets="none"` so they won't be referenced transitively by any dependant projects.
I've checked the set of assemblies included in the VSIX has not changed.